src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
```

Comment:
is it necessary to use all styles from layout_catalog? i thought we got a mock-up inside layout_catalog_grid and we should just make it work correctly